subsys/net/ip/tcp.c

```diff
@@ -1344,3 +1412,1160 @@ int net_tcp_parse_opts(struct net_pkt *pkt, int opt_totlen,
 	NET_ERR("Invalid TCP opt: %d len: %d", opt, optlen);
 	return -EINVAL;
 }
+
+int tcp_hdr_len(struct net_pkt *pkt)
```

Comment:
This function used to be static; if it's not any longer, please call it `net_tcp_hdr_len()`.

Reply:
This function became public, as it was moved to tcp.c. With the rest of the stack moving to tcp.c, this function can again be made static.

Reply:
Sorry, no. tcp_hdr_len() is defined in tcp.c, but the usage is actually in net_context.c.